### ISSUE TITLE
MultiColor Stroke Around Text

### ISSUE THREAD

**Issue by zelin** [m5/MagicTextView#21]
Is multicolor stroke around text possible even by modifying your code?

**Comment by m5** [comment]
I don't think this is going to be a good solution for you. You could do multiple strokes with different colors and stroke sizes, but it will only allow you to do different contours of colors.

**Comment by zelin** [comment]
well i want to bold the text a big . so best way is to add a stroke to text with same color and then add another stroke however i am not sure how can this be done using paint of textView.
